### DISCUSSION TITLE
版面分析文本块未识别

### DISCUSSION THREAD

**Discussion by PureWaterCatt** [PaddlePaddle/PaddleOCR#13133] [Q&A]
#### 问题描述 / Problem Description
我将有问题的PDF页面单独拿出来，如下图展示，部分文字并未识别出来。暂未测直接ocr的方式。

#### 运行环境 / Runtime Environment
- main分支

#### 复现代码 / Reproduction Code
`im_show = draw_structure_result(img, results, font_path='../../doc/fonts/simfang.ttf')
    im_show = Image.fromarray(im_show)
    im_show.save('./result/result_{}.jpg'.format(index))`

#### 完整报错 / Complete Error Message
![result_0](https://github.com/PaddlePaddle/PaddleOCR/assets/169960786/0f993ab4-13fb-48a8-a8d2-4b4382fd5a15)
![result_1](https://github.com/PaddlePaddle/PaddleOCR/assets/169960786/804ae94d-be17-4dca-ac3d-2fc9774707d7)

#### 附件 / Appendix
[test1.pdf](https://github.com/user-attachments/files/15543596/test1.pdf)
[test2.pdf](https://github.com/user-attachments/files/15543597/test2.pdf)

**Comment by PureWaterCatt** [comment]
测试多个类似的pdf文档，效果也是识别不到审核结论这个位置
效果：
![result_0](https://github.com/PaddlePaddle/PaddleOCR/assets/169960786/704543b8-96dc-4134-8c44-0b8fdc9504fa)
![result_0](https://github.com/PaddlePaddle/PaddleOCR/assets/169960786/1fd8c4fd-f71b-4854-9bd4-38f0343c49d3)

附件：
[test11.pdf](https://github.com/user-attachments/files/15544099/test11.pdf)
[test12.pdf](https://github.com/user-attachments/files/15544115/test12.pdf)

**Reply by zhangtingyun** [reply]
方便问一下你的模型参数吗？

**Reply by PureWaterCatt** [reply]
我直接使用的推理模型，没有进行其他操作

**Reply by bla-l** [reply]
请问问题解决了吗

**Comment by GreatV** [comment]
确实会有一些没识别到，后面我检查一下问题。

![result](https://github.com/PaddlePaddle/PaddleOCR/assets/17264618/2242e33f-34a7-4d7b-9d73-c61a08ac21e8)

![result_test2_1](https://github.com/PaddlePaddle/PaddleOCR/assets/17264618/aafba9ea-8f1b-4492-80cc-529906c41ffc)

**Reply by bla-l** [reply]
有结论了吗，我也遇到这种情况了

**Comment by SWHL** [comment]
这个问题不像是代码bug，更像是模型效果不好导致的。
